<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-VEMALAM" ignored="0" name="autotest-mainTest" tests="2" failures="1" timestamp="20 二月 2020 07:17:49 GMT" time="0.078" errors="0">
  <testcase name="success_test_01" time="0.023" classname="testCase.HelloTestNG"/>
  <testcase name="success_test_02" time="0.001" classname="testCase.HelloTestNG">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
	at testCase.HelloTestNG.success_test_02(HelloTestNG.java:20)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- success_test_02 -->
</testsuite> <!-- autotest-mainTest -->
